### PR TITLE
Update Microsoft.Net.Native.Compiler dependency

### PR DESCRIPTION
**Pull request by zamont**
Also update the property to store that dependency value.

### CODE REVIEW

**Comment by zamont** [comment]
PTAL @joperezr @weshaggard

File: dependencies.props [hunk 1]
```diff
@@ -20,6 +20,7 @@
     <PlatformPackageVersion>2.1.0-preview1-25324-02</PlatformPackageVersion>
     <MicrosoftNETCoreRuntimeCoreCLRPackageVersion>2.1.0-preview1-25413-01</MicrosoftNETCoreRuntimeCoreCLRPackageVersion>
     <MicrosoftNETCoreRuntimeJitPackageVersion>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</MicrosoftNETCoreRuntimeJitPackageVersion>
+    <MicrosoftNetNativeCompilerPackageVersion>2.0.0-preview-25429-00</MicrosoftNetNativeCompilerPackageVersion>
```
Comment (joperezr) [inline dependencies.props]: Who builds this package?

Comment (zamont) [inline dependencies.props]: This is produced internally by the .NET Native team. It is unfortunately not automated yet.

Comment (zamont) [inline dependencies.props]: We then push the package to MyGet / NuGet. So this preview build is sitting on MyGet right now.

**Pull request merged**